configure.ac

```diff
@@ -70,6 +70,19 @@ GX_C__GENERIC
 AS_IF([test $gx_cv_c__Generic = no],
   AC_MSG_ERROR([The C compiler does not support the generic selection C-11 standard.  Please use a C-11 compliant compiler.])
 )
```

Comment:
@rem1776  @mlee03 
I can compile on the gpubox with nvc and without the configure option "--with-openacc". But if that option is added , then there are compilation warnings and  failures concerning the _Generic fabs declared in mosaic_util.h. I tried using the -std=c99, -std=c11, and -std=c17 as I have tried with other compilers, but in this case I get compilation messages like this:
```
"../../../tools/libfrencutils/create_xgrid.c", line 2296: warning: function "_Generic" declared 
     implicitly [implicit_func_decl]
```
Did anyone get past this problem in an acceptable way ?

Reply:
I get compilation errors too, it seems like the user defined CFLAG (and hence the -std=c11) gets overwritten

Reply:
I was wondering why I wasn't seeing this but looks like you figured it out, I was configuring using CC="nvc -std=c11" and it seemed to run fine cause it wasn't getting rid of the flag. That should be fixed now cause it'll add the openacc flags where needed instead of just setting them in the configure script.